index.html

```diff
@@ -2118,6 +2118,11 @@ <h2>Resource Integrity</h2>
 <a href="#multibase-0">Multibase</a>-encoded <a href="#multihash">Multihash</a>
 values.
         </p>
+        <p>
+`digestMultibase` is expected to be added to the context of documents that need to refer to
+other resources and include an associated content integrity hash. For example, the Verifiable
+Credential Data Model v2.0 context (`https://www.w3.org/ns/credentials/v2`) includes it.
```

Comment:
Identify which role the expectation applies to; "JSON-LD context authors":

```suggestion
JSON-LD context authors are expected to add `digestMultibase` to contexts that will 
be used in documents that refer to other resources and include an associated 
content integrity hash. For example, the Verifiable Credential Data Model v2.0 
context (`https://www.w3.org/ns/credentials/v2`) includes it.
```

Reply:
Looks good to me.